client/src/components/SupportTools/SupportTools.tsx

```diff
@@ -393,16 +393,23 @@ const Audit = ({ electionId }: { electionId: string }) => {
 
   if (!election.isSuccess) return null
 
-  const { auditName, auditType, jurisdictions, rounds } = election.data
+  const { id, auditName, auditType, jurisdictions, rounds } = election.data
 
   return (
     <Column>
       <H2>{auditName}</H2>
```

Comment:
While I was here, I noticed that the audit name wraps with the column, which doesn't look great for longer audit names. Maybe the `H2` could be outside of the `Column`?

Reply:
Good catch - will tweak 🛠️